config/default/constants.js

```diff
@@ -123,4 +123,12 @@ module.exports = {
 	TOTAL_AMOUNT: '10000000000000000',
 	UNCONFIRMED_TRANSACTION_TIMEOUT: 10800, // 1080 blocks
 	EXPIRY_INTERVAL: 30000,
+	CACHE: {
```

Comment:
I reckon this constants should be part of the cache component.
I suggest to create a folder for each component inside `components` and move this constants there. Also it could be exported along with the component.

Something like:

```
/components
    cache/
        constants.js
        cache.js
        index.js
    index.js
```